lua/damagelogs/shared/lang/simplified_chinese.lua

```diff
@@ -0,0 +1,413 @@
+    Finished = "完成",
+    ReportIsntFinished = "这份举报还未完成！",
+    HasSetConclusion = "已对举报做出结论",
+    TheReportedPlayer = "被举报的玩家",
+    HasAnsweredReport = "%s 已回答举报 #%s !",
+    HasCanceledReport = "%s 已取消举报 #%s。",
+    TheReport = "举报",
+    HasCanceledByVictim = "已被受害者取消！",
+    NoMercy = "举报中，受害者没有原谅攻击者",
+    DidNotForgive = "%s 没有在举报 #%s 中原谅 %s !",
+    C4Armed = "%s [%s] 启动了 %s 的C4。",
+    C4Disarmed = "%s [%s] 拆除了 %s 的C4 %s 成功。",
+    C4PickedUp = "%s [%s]拾起了 %s 的C4。",
+    C4Planted = "%s [%s] 安装或丢下了一个C4。",
+    C4Destroyed = "%s [%s] 破坏了 %s 的C4。",
+    C4Exploded = "%s [%s] 的C4已爆炸。",
+    with = "使用",
+    without = "未使用",
+    enabled = "启用",
+    disabled = "禁用",
+    AutoSlain = "%s [%s] 已自动被杀。",
+    DisguiserAct = "%s [%s] %s 他们的伪装者",
+    Teleported = "%s [%s] 已传送",
+    DisguiserSpam = "%s [%s] 正在刷他们的伪装者。伪装日志将停止。",
+    TrapActivated = "%s 已激活一个叛徒陷阱： %s",
+    RoundBought = "%s 已购买一个叛徒回合",
+    NadeThrown = "%s [%s] 投掷了 %s",
+    UsedCredits = "%s [%s] %s %s 信用%s",
+    received = "收到",
+    used = "使用",
+    BodyIdentified = "%s [%s] 已确认 %s [%s] 的尸体",
+    VictimShotFirst = "受害者可能先开火（查看伤害信息部分以获取更多信息）！",
+    HasDamaged = "%s [%s] 已对 %s [%s] 造成 %s",
+    HPWeapon = " HP 使用 %s",
+    sbody = "的尸体",
+    DNAretrieved = "%s [%s] 已从 %s 中获取 %s [%s] 的DNA",
+    FallDamage = "%s [%s] 跌落并丢失 %s HP",
+    AfterPush = " 在被 %s [%s] 推后",
+    HealthStationHeal = "%s [%s] 使用 %s 的医疗站治疗了 %s HP",
+    HasKilled = "%s [%s] 用 %s 杀死了 %s [%s]",
+    UnknownWeapon = "未知武器",
+    SomethingKilled = "<某物/世界> 杀死了 %s [%s]",
+    Radar = "一个雷达",
+    Armor = "一套防弹衣",
+    Disguiser = "一个伪装者",
+    HasBought = "%s [%s] 购买了 %s",
+    HasUsed = "%s [%s] 已使用设备 %s (%s)",
+    CrowbarSwung = "%s - %s 挥动了他们的撬棍",
+    HasShot = "%s - %s 使用了 %s",
+    HasPushed = "%s - %s 用撬棍推了 %s",
+    RadioUsed = "%s [%s] 使用了他们的无线电： %s%s",
+    CorpseOf = "尸体属于 ",
+    traitor = "叛徒",
+    detective = "侦探",
+    disconnected = "已断开连接",
+    innocent = "无辜",
+    CurrentRoundSelected = "当前选定的回合： ",
+    CopyLines = "复制行",
+    Copy = "复制",
+    Lines = "行",
+    FullDisplay = "全屏显示",
+    SteamIDof = "SteamID 为 ",
+    SteamID = "SteamID",
+    Close = "关闭",
+    ShowDeathScene = "展示死亡场景",
+    ShowDamageInfos = "展示伤害信息",
+    EmptyLogsFilters = "空的日志。请检查您的筛选条件！",
+    EmptyLogs = "空的日志",
+    RoundEnded = "回合结束",
+    DamageInfosBetween = "%s 和 %s 之间的 %s 到 %s 的伤害信息。",
+    PlayerInformation = "玩家信息",
+    SelectDate = "选择日期：",
+    SelectRound = "选择回合：",
+    RoundsBetween = "%s:00 和 %s:00 之间的回合",
+    SelectRoundToLoad = "选择需要加载的回合",
+    PleaseSelectRound = "请选中一个回合！",
+    LoadLogsSelected = "加载选定回合的日志",
+    OldLogs = "旧日志",
+    ShotLogs = "开火日志",
+    ShotEvent = "开火事件",
+    CopySteamID = "复制SteamID64",
+    SpectateAttacker = "观看攻击者",
+    SpectateVictim = "观看受害者",
+    FreeMode = "自由模式",
+    EnableMouse = "注意：按C键启用鼠标。",
+    EnableSlowMotion = "启用慢动作",
+    NoWeapon = "<无武器>",
+    scorpse = "的尸体",
+    TakeAction = "采取行动",
+    HaventReported = "您还没有报告任何人！没有报告可以查看。",
```

Comment:
This key doesn't exist in the other languages or the code?

Reply:
Indeed no row 224 didn't pay attention :D